resources/views/platform/components/game/link-buttons/index.blade.php

```diff
@@ -100,4 +101,13 @@
             Set Requestors
         </x-game.link-buttons.game-link-button>
     @endif
+
+    @if ($canSeeSuggestedGames)
+        <x-game.link-buttons.game-link-button
+            icon="🕹️"
+            href="{{ route('game.suggest-for-game', $gameId) }}"
+        >
+            Find something similar to play
```

Comment:
nit: I can't help but notice the casing on this makes it stick out from the rest of the buttons.

![Screenshot 2024-01-15 at 6 15 41 PM](https://github.com/RetroAchievements/RAWeb/assets/3984985/be53fbb3-1521-478f-8cdf-7ce48c7e6baf)

Because the button is so close to the "Similar Games" header, I wonder if users might find this ambiguous or duplicative.

![Screenshot 2024-01-15 at 6 18 12 PM](https://github.com/RetroAchievements/RAWeb/assets/3984985/27a032dd-28aa-45ee-9893-6ea252a34255)

The other buttons all seem to be nouns. Maybe something like "Suggested Games" would work better as a label here?

Reply:
It's a tough one for sure. I notice in latest that the capitalization is now at least consistent with the rest of the buttons - that's probably good enough for now.